packages/next/src/server/app-render/metadata-insertion/create-server-inserted-metadata.tsx

```diff
@@ -16,21 +11,8 @@ document.querySelectorAll('body link[rel="icon"], body link[rel="apple-touch-ico
 
 export function createServerInsertedMetadata(nonce: string | undefined) {
   let inserted = false
-  const setMetadataResolver = (_resolver: MetadataResolver): void => {}
 
   return {
-    ServerInsertedMetadataProvider: ({
-      children,
-    }: {
-      children: React.ReactNode
-    }) => {
-      return (
-        <ServerInsertedMetadataContext.Provider value={setMetadataResolver}>
-          {children}
-        </ServerInsertedMetadataContext.Provider>
-      )
-    },
-
     async getServerInsertedMetadata(): Promise<string> {
```

Comment:
since we are only returning one thing now maybe just return it rather than an object?